en/sdk/index.md

```diff
@@ -1,19 +1,19 @@
-# Integrate SDKs
+# Integrating SDKs
+**Cocos Creator** easily integrates with other **SDKs**. 
 
-The SDK can be integrated in the following ways:
-
-- The AnySDK can be integrated through the **Menu bar -> Panel -> Service** panel.
+### AnySDK
+**AnySDK** can be integrated through the **Menu bar -> Panel -> Service** panel.
 
   ![](cocos-services/cocos_services.png)
   
-  For more information, please refer to:
-    - [Cocos Services](sdk/cocos-services.md)
-    - [AnySDK](sdk/anysdk.md)
+Please refer to the [Cocos Services](sdk/cocos-services.md) and [AnySDK](sdk/anysdk.md) documentation for details.
 
-- You can integrate Facebook Live Video and Audience Network by ticking the **Menu bar -> Project-> Project Settings -> Service -> Facebook Service**. Please refer to [Facebook Live Video & Audience Network](fb-an-and-live.md) for details.
+### Facebook Live Video and Audience Network
+**Facebook Live Video and Audience Network** can be integrated by selecting the **Menu bar -> Project-> Project Settings -> Service -> Facebook Service**. Please refer to [Facebook Live Video & Audience Network](fb-an-and-live.md) documentation for details.
```

Comment:
**Facebook Live Video and Audience Network** -> **Facebook Live Video** and **Audience Network**